src/safeposix/syscalls/net_calls.rs

```diff
@@ -686,11 +667,11 @@ impl Cage {
             return syscall_error(Errno::EBADF, "cleanup socket", "invalid file descriptor");
         }
 
-        //We have to take this out of the match because the fdtable already has a mutable borrow
-        //which means that I can't change it with a remove until after the fdtable mutable borrow is finished from the match statement
+        //We have to take this out of the match because the self.filedescriptortable already has a mutable borrow
```

Comment:
does this still make sense?

Reply:
The word mutable does not but otherwise yes it does, but I can restructure it to be better anyway.